alpakka-sample-file-to-elasticsearch/src/main/java/samples/javadsl/Main.java

```diff
@@ -1,19 +1,19 @@
 package samples.javadsl;
 
-import akka.Done;
-import akka.NotUsed;
-import akka.actor.typed.ActorSystem;
-import akka.actor.typed.javadsl.Behaviors;
-import akka.japi.Pair;
-import akka.stream.KillSwitches;
-import akka.stream.UniqueKillSwitch;
-import akka.stream.alpakka.elasticsearch.*;
-import akka.stream.alpakka.elasticsearch.javadsl.ElasticsearchFlow;
-import akka.stream.alpakka.elasticsearch.javadsl.ElasticsearchSource;
-import akka.stream.alpakka.file.DirectoryChange;
-import akka.stream.alpakka.file.javadsl.DirectoryChangesSource;
-import akka.stream.alpakka.file.javadsl.FileTailSource;
-import akka.stream.javadsl.*;
+import org.apache.pekko.Done;
```

Comment:
the preferred import style is

```
import org.apache.pekko
import pekko.NotUsed
...
```

Reply:
Resolved: [3cfb2fa](https://github.com/apache/incubator-pekko-connectors-samples/pull/2/commits/3cfb2fa544e45c61754e160b1563fa9748e26327)